src/templates/registration/base.html

```diff
@@ -0,0 +1,79 @@
+
+<!DOCTYPE html>
+<html>
+<head>
+    <!-- Standard Meta -->
+    <meta charset="utf-8" />
+    <meta http-equiv="X-UA-Compatible" content="IE=edge,chrome=1" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0, maximum-scale=1.0">
+
+    <!-- Site Properties -->
+    <title>Login Example - Semantic</title>
```

Comment:
Fix title?

Can we put this file in `templates/base_simple.html` or something, and put `{% extends "base_simple.html" %} in `registration/base.html` 

That way, when we want simple 1 page things with nothing else?